src/ScheduleFunctions.cpp

```diff
@@ -401,6 +400,13 @@ Stmt build_produce(const map<string, Function> &env, Function f, const Target &t
 
         const string &extern_name = f.extern_function_name();
 
+        // We need to generate crops of the input and output buffers
+        // if the extern stage has some non-extern loops.
+        bool needs_crops = false;
```

Comment:
Better to initialize needs_crops as store_level != compute_level and simplify conditions below?

Reply:
I actually did that at first, but they're different (f.store_level != f.compute_level, vs. input.store_level != input.compute_level, where input is a loop variable). That actually seemed a bit suspicious to me...